src/main/java/com/epam/emotionalhelp/controller/dto/QuestionRequestDto.java

```diff
@@ -0,0 +1,16 @@
+package com.epam.emotionalhelp.controller.dto;
+
+import com.epam.emotionalhelp.model.Emotion;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
```

Comment:
Can we consider [@Value](https://projectlombok.org/features/Value) annotation instead?